ecosystem/typescript/sdk_v2/src/transactions/authenticator/transaction.ts

```diff
@@ -181,3 +184,33 @@ export class TransactionAuthenticatorFeePayer extends TransactionAuthenticator {
     return new TransactionAuthenticatorFeePayer(sender, secondary_signer_addresses, secondary_signers, fee_payer);
   }
 }
+
+export class TransactionAuthenticatorSecp256k1 extends TransactionAuthenticator {
+  public readonly public_key: Secp256k1PublicKey;
+
+  public readonly signature: Secp256k1Signature;
+
+  /**
+   * Transaction authenticator Secp256k1 for a single signer transaction
```

Comment:
```suggestion
   * A transaction authenticator for a Secp256k1 keypair
```

or something like that..? we should just go back to this when we review the documentation at the end